_inc/client/components/jetpack-notices/index.jsx

```diff
@@ -168,10 +168,11 @@ export class UserUnlinked extends React.Component {
 			return (
 				<div className="jp-unlinked-notice">
 					<JetpackBanner
-						title={ __( 'Connect your account to get the most out of Jetpack' ) }
-						callToAction={ __( 'Connect to WordPress.com' ) }
+						title={ __( 'Jetpack is powering your site, but to access all of its features you’ll need to create a Jetpack account.' ) }
```

Comment:
I think it would be good to have a copy review for this. I think it all looks pretty dang good, though. Nice work!

Reply:
@Automattic/editorial can you help with this, please?